spec/migrations/helpers/migration_mixin.rb

```diff
@@ -0,0 +1,35 @@
+module MigrationMixin
```

Comment:
This mixin looks _interesting_... I've checked out the code and turned it into a **shared context**. Looks less hacky, I would say: https://github.com/sap-contributions/cloud_controller_ng/commit/ad8441b3225449a94fbc9879b8474324a7adb058